packages/airnode-validator/src/v2/receipt/receipt.ts

```diff
@@ -0,0 +1,33 @@
+import { z } from 'zod';
+import { cloudProviderSchema } from '../config';
+import { SchemaType } from '../types';
+
+// TODO: Implement old validator rules
```

Comment:
Do you mean improving the types to enforce strings conforming to a certain regex pattern as well?

Reply:
Yeah, I will implement those - but Zod has already helper functions for most of what we need -  https://github.com/colinhacks/zod#strings

Reply:
The good part again is that I trust they probide a pretty good error messages that have been reviewed an bikeshed by number of people :)